contract/models/contract_modification.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ContractModification(models.Model):
+
+    _name = 'contract.modification'
+    _description = 'Contract Modification'
+    _order = 'date desc'
+
+    date = fields.Date(
+        required=True,
+        string='Date'
+    )
+    description = fields.Text(
+        required=True,
+        string='Description'
+    )
+    contract_id = fields.Many2one(
+        string='Contract',
+        comodel_name='contract.contract',
+        required=True,
+        ondelete='cascade',
+        index=True
+    )
+    sent = fields.Boolean(
+        string='Mail notify',
+        default=False
+    )
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        record = super(ContractModification, self).create(vals_list)
```

Comment:
Multiple records can be returned
```suggestion
        records = super(ContractModification, self).create(vals_list)
```